src/assets/scss/_navigation.scss

```diff
@@ -0,0 +1,20 @@
+#header-nav{
+
+  li{
+    position: relative;
+  }
+
+  li:after{
```

Comment:
Nit. Be consistent in your coding style: 

1. Indent with 4 spaces
2. Leave a space before `{`

Reply:
Do you know any good sass/scss linter ? I found this [one](https://atom.io/packages/linter-sass-lint) but you have to create a `.sass-lint.yml` in the project's root folder. Is that ok? Should I commit it?

Reply:
Yes, please add `.sass-lint.yml`. Of course we'll want to use the `sass-lint` package, not the respective atom package.